app/src/ui/remove-repository/confirm-remove-repository.tsx

```diff
@@ -29,6 +42,7 @@ export class ConfirmRemoveRepository extends React.Component<
   }
 
   public render() {
+    const trashName = __DARWIN__ ? 'Trash' : 'Recycle Bin'
```

Comment:
This concept is duplicated should it be made into a base class?

Reply:
I'd recommend a function that can be exported and imported, like this:

https://github.com/desktop/desktop/blob/4da77727b8430cd433a8ccab5f312eb5d464b0d3/app/src/ui/lib/context-menu.ts#L6-L8